src/testsets.jl

```diff
@@ -173,14 +162,19 @@ mutable struct TestRelTestSet <: AbstractTestSet
 end
 
 function record(ts::RAITestSet, child::TestRelTestSet)
-    tc = JUnitTestCase(child.dts)
+    # Strip additional `file:line` info, so names are robust to movement in files.
+    # This currently relies on `" @ "` not appearing in the `name` of the `@test_rel`.
+    name = chopsuffix(child.dts.description, r" @ .*")
```

Comment:
would we make this regex a little tighter instead of hoping that `@` isn't in the test name?

Reply:
yeah... i was trying to get away without having to think about regex 😂 

i've added a more specific one -- could you take a look?